docs/_docs/features/debugger.md

```diff
@@ -6,7 +6,7 @@ permalink: /docs/features/debugger/
 ---
 
 One of the key features of Nuclide is its multiple-language debugging support. The Nuclide Debugger
-is provided within the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
+is provided with the familiar [Chrome DevTools](https://developer.chrome.com/devtools) interface.
```

Comment:
Is this true any longer given the new interface? I know there is a setting to go back to the old chrome dev interface, but in general, the new interface is a bit different now, right?